src/metrics/moves.ts

```diff
@@ -106,3 +106,106 @@ export class PieceLevelMoveInfoMetric implements Metric {
     return averagesMap;
   }
 }
+
+export class MiscMoveFactMetric implements Metric {
+  enPassantMoves: number;
+  knightHops: UAPMap<{ count: number }>; // the number of times a piece is hopped over by knights
+
+  constructor() {
+    this.clear();
+  }
+
+  clear(): void {
+    this.enPassantMoves = 0;
+    this.knightHops = createUAPMap({ count: 0 });
+  }
+
+  processGame(
+    game: { move: PrettyMove; board: Piece[] }[],
+    metadata?: string[]
+  ) {
+    // since a knight can take 2 paths to hop over a piece we ensure that it is forced to hop over at least one piece.
+    // When it must hop we assume it hops all pieces so that the result is deterministic
```

Comment:
adding to readme

Reply:
does that mean this is no longer true? this was in our readme previously: When a knight "hops" its path is chosen based on if the current move is even or odd: i.e. two squares first if even, one if odd (This is done to ensure consistency in metrics across runs)

Reply:
if no longer true pls remove from readme

Reply:
I wanted something deterministic. Either approach would be. Do you have a preference

Reply:
no strong preference, can do random for now